app/assets/stylesheets/components/_navbar.scss

```diff
@@ -0,0 +1,55 @@
+.navbar-toggle {
+  display: none;
+
+  // @include media($screen-tablet) {
+  // }
+}
+
+.navbar-brand {
+  font-size: 2rem;
+  color: $black-color;
+  margin: 0;
+  background: url("#{$images-path}/ruby-logo.png") no-repeat left;
+  padding-left: 65px;
+  line-height: 65px;
+  display: block;
+}
+
+.navbar {
+  position: fixed;
```

Comment:
Properties should be ordered background-color, border-bottom, min-height, padding, position, top, width, z-index